pages/content/amp-dev/documentation/courses/beginning-course/thinking-in-components@pt_BR.md

```diff
@@ -0,0 +1,265 @@
+---
+$title: Pensando em componentes
+$order: 4
+leveled: true
+---
+
+[filter level="beginner"]
+## Adicionando recursos ao nosso site
+
+Até agora, convertemos nosso site HTML básico em um site AMP básico. Há apenas um erro de validação restante em nosso site, e vamos corrigi-lo usando um componente `<amp-img>` em vez de uma tag `<img>`. Conforme corrigimos esse último erro de validação, aprenderemos o que são os componentes de AMP, por que algumas tags HTML foram substituídas ou não permitidas em AMP e como adicionar componentes ao nosso site.
+
+Depois disso, é hora de adicionar funcionalidades adicionais ao nosso site. Para completar a versão inicial da página inicial da Chico’s Cheese Bike, vamos adicionar algum conteúdo de marketing adicional. A equipe decidiu adicionar um vídeo no YouTube sobre como fazer nossas bicicletas de queijo, um carrossel de imagens de nossos vários produtos de bicicletas de queijo e alguns links de mídia social que ajudarão os usuários a compartilhar nosso site com suas redes sociais favoritas.
+
+Pode parecer assustador adicionar tantas peças ao nosso site tão rapidamente. Precisaríamos criar HTML, CSS e JavaScript para cumprir muitos dos requisitos para os recursos que queremos adicionar (como alterar o slide ativo do carrossel). Depois, teríamos que considerar como fazer todo o site ter um bom desempenho.
+
+Mas essa é a beleza do AMP. Com AMP, não precisamos nos preocupar com todos esses detalhes! Os autores da biblioteca AMP nos forneceram blocos de construção que nos fornecem esses recursos e ajudam a cuidar de qualidades como desempenho, acessibilidade e segurança. Esses blocos são chamados de **componentes** e são a chave para criar sites de sucesso com AMP.
+
+## O que são Web Components?
+
+Os componentes são blocos de construção para a web. Eles representam a combinação de estrutura (HTML), estilo (CSS) e comportamento (JavaScript) com uma interface que facilita o uso em seu site e o compartilhamento com outras pessoas. Os componentes têm:
+
+- Um nome (por exemplo, `<amp-img>`) usado como o nome da tag para identificar o componente.
+
+- Atributos personalizados que alteram o comportamento, estilo ou conteúdo de um componente (como `width`,` height`, `src` e`attribution`).
```

Comment:
can you insert a space before "attribution"?

Reply:
Done